lib/database_dumper.rb

```diff
@@ -1205,6 +1203,259 @@ def self.actions_to_column_sanitizers(columns_by_action)
     }.freeze,
   }.freeze
 
+  V2_RESULTS_SANITIZERS = {
+    "results" => {
+      column_sanitizers: actions_to_column_sanitizers(
+        copy: %w[
+          pos
+          best
+          average
+        ],
+        fake_values: {
+          "competition_id" => "competition_id",
+          "event_id" => "event_id",
+          "round_type_id" => "round_type_id",
+          "person_name" => "person_name",
+          "person_id" => "person_id",
+          "format_id" => "format_id",
+          "regional_single_record" => "regional_single_record",
+          "regional_average_record" => "regional_average_record",
+          "person_country_id" => "country_id",
+        }.freeze,
+      ),
+    }.freeze,
+    "result_attempts" => {
+      column_sanitizers: actions_to_column_sanitizers(
+        copy: %w[
+          id
+          value
+          attempt_number
+          result_id
+          created_at
+          updated_at
+        ],
+      ),
+    }.freeze,
+    "ranks_single" => {
+      column_sanitizers: actions_to_column_sanitizers(
+        copy: %w[
+          best
+        ],
+        fake_values: %w[
+          person_id
+          event_id
+          world_rank
+          continent_rank
+          country_rank
+        ],
+      ),
+    }.freeze,
+    "ranks_average" => {
+      column_sanitizers: actions_to_column_sanitizers(
+        copy: %w[
+          best
+        ],
+        fake_values: %w[
+          person_id
+          event_id
+          world_rank
+          continent_rank
+          country_rank
+        ],
+      ),
+    }.freeze,
+    "round_types" => {
+      column_sanitizers: actions_to_column_sanitizers(
+        copy: %w[
+          id
+          final
+          name
+          rank
+        ],
+        fake_values: %w[
+          cell_name
+        ],
+      ),
+    }.freeze,
+    "events" => {
+      column_sanitizers: actions_to_column_sanitizers(
+        copy: %w[
+          id
+          format
+          name
+          rank
+        ],
+        fake_values: {
+          # Copy over column to keep backwards compatibility
```

Comment:
Now we're introducing a major version break we don't have to worry about any backwards compatibility anymore